freva-rest/src/databrowser_api/core.py

```diff
@@ -1208,9 +1277,22 @@ async def add_user_metadata(
             "fs_type": self.fs_type
         } | fwrites.get("facets", {})
         await self._ingest_user_metadata(user_metadata)
+
         logger.info(
             "Ingested %d files into Solr and MongoDB", self.total_ingested_files
         )
+        if self.total_ingested_files == 0:
+            status_msg = (
+                "No data was added to the databrowser. "
+                "(No files ingested into Solr and MongoDB)"
+            )
+        else:
+            status_msg = (
+                f"Your data has been successfully added to the databrowser. "
+                f"(Ingested {self.total_ingested_files} files into Solr and MongoDB)"
```

Comment:
You can just have it saying: f"{self.total_ingested_files} have been successfully added to the databrowser."